packages/mgt-chat/src/components/ChatListItem/ChatListItem.tsx

```diff
@@ -155,14 +176,30 @@ export const ChatListItem = ({ chat, myId, onSelected }: IMgtChatListItemProps &
     return String(timestamp);
   };
 
-  const getDefaultProfileImage = () => {
+  const getDefaultProfileImage = (chatObj: Chat) => {
     // define the JSX for FluentUI Icons + Styling
     const oneOnOneProfilePicture = <ChatListItemIcon chatType="oneOnOne" />;
     const GroupProfilePicture = <ChatListItemIcon chatType="group" />;
 
+    const other = chatObj.members?.find(m => (m as AadUserConversationMember).userId !== myId);
+    const otherAad = other as AadUserConversationMember;
+    let iconId: string | undefined;
     switch (true) {
       case chat.chatType === 'oneOnOne':
-        return oneOnOneProfilePicture;
+        if (!otherAad) {
+          iconId = myId;
+        } else {
+          iconId = otherAad?.userId as string;
+        }
+        return (
+          <Person
```

Comment:
External user pics are not showing up. Looking into it for more details.

Reply:
External user pics won't work without a major code re-write; however, the updated code I have submitted to the PG should address setting a default icon. The code I sent you in chat should address how to set the template for that default icon.